.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -33,10 +39,14 @@ jobs:
           - qemu-tdx-experimental
           - rootfs-image
           - rootfs-initrd
+          - rootfs-initrd-mariner
           - rootfs-initrd-sev
           - shim-v2
           - tdvf
           - virtiofsd
+        exclude:
+          - stage: release
+            asset: cloud-hypervisor-glibc
```

Comment:
do you want to exclude `rootfs-initrd-mariner` for `release` as well?

Reply:
@fidencio Thoughts?

Reply:
Talked with Fabiano and will keep this as it does no harm

Reply:
Yep, I do not have any strong feelings on keeping it, or dropping it.
If someone at some point complains the image becomes too big, then we'll re-evaluate what to do.  For now, let's keep it in.